src/View/Helper/DocBlockHelper.php

```diff
@@ -230,6 +231,14 @@ public function columnTypeToHintType(string $type): ?string
                 }
 
                 return '\Cake\I18n\Time';
+
+            default:
+                if (str_starts_with($type, 'enum-')) {
+                    $dbType = TypeFactory::build($type);
```

Comment:
@LordSimal Won't this throw an exception if the type isn't found? Is that ok for a catch-all?

Reply:
Indeed, we might wanna use try catch

Reply:
I am not really sure this situation is possible.
Getting to this case where the cake type string starts with `enum-` (basically) requires it being generated by
```
EnumType::from(MyEnum::class)
```
So that DBType needs to be present in the TypeFactory